fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -591,6 +596,14 @@ protected void addResourceHttpHeaders(final FedoraResource resource) {
         addResourceHttpHeaders(resource, false);
     }
 
+
+    protected void addRegisterHeaders() {
+        if (fedoraPropsConfig.getFcrepoRegsiterApiBanner() == true) {
+            servletResponse.addHeader("USER-REGISTRATION", REGISTER_REPO_TEXT + " " + REGISTER_REPO_LINK);
+            servletResponse.addHeader("Link", "<" + REGISTER_REPO_LINK + ">; rel=\"user-regisration\"");
```

Comment:
To be valid against the rules of RFC 8288 for [extension relations](https://www.rfc-editor.org/rfc/rfc8288#section-2.1.2) the `rel` needs to be in the form of a URI. Sort of like how we do for [external content](https://wiki.lyrasis.org/display/FEDORA6x/External+Content)